### PR TITLE
Update README.md: cache.py is not here

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ couple of `fancy` features.
 - __daemon.py__ : Daemon class written by Sander Marechal (http://www.jejik.com) to start a python script as a daemon.
 - __missing.py__: Small functions and tools that are commonly used but not
   available in the Python (2.x) API.
-- __cache.py__ : File cache to store pickled data identified by a key accompanied by a timestamp,
+- ~~__cache.py__ : File cache to store pickled data identified by a key accompanied by a timestamp.~~ Was moved to vsc-utils
```
Comment (boegel) [inline README.md]: fix casing (`was` instead of `Was`)

provide a reference to `vsc-utils`, i.e. https://github.com/hpcugent/vsc-utils

Comment (JensTimmerman) [inline README.md]: The `Was` comes right after a `.` so I believe the capitalisation is correct? Please let me know what rule I'm missing if this is incorrect.

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ couple of `fancy` features.
 - __daemon.py__ : Daemon class written by Sander Marechal (http://www.jejik.com) to start a python script as a daemon.
 - __missing.py__: Small functions and tools that are commonly used but not
   available in the Python (2.x) API.
-- __cache.py__ : File cache to store pickled data identified by a key accompanied by a timestamp,
+- ~~__cache.py__ : File cache to store pickled data identified by a key accompanied by a timestamp.~~ Was moved to [vsc-utils](https://github.com/hpcugent/vsc-utils)
```
Comment (boegel) [inline README.md]: Grammar-wise: no subject in the `Was` sentence, if you need a reason to change it.

Comment (boegel) [inline README.md]: suggestion:

```
; was moved to ...
```

Comment (JensTimmerman) [inline README.md]: fixed

**Pull request merged**